browser/js/dashboard/gridbox/widgetCtrl.js

```diff
@@ -1,5 +1,7 @@
-app.controller('WidgetCtrl', ['$scope', '$uibModal',
-    function($scope, $uibModal) {
+
+app.controller('WidgetCtrl', ['$scope', '$modal', '$controller', '$rootScope', 'WidgetSettingsFactory', 'DashboardFactory', '$interval', '$uibModal',
```

Comment:
If you're using this "long hand" for injecting dependencies to avoid minification woes, you might [check out `ng-annotate`](https://github.com/olov/ng-annotate).